cpp/src/gandiva/precompiled/bitmap.cc

```diff
@@ -38,11 +38,7 @@ FORCE_INLINE
 void bitMapSetBit(unsigned char* bmap, int position, bool value) {
```

Comment:
NB the function name and variable names in this function don't conform to the Arrow style guide

Reply:
Is there is any documents that explains the codestyle that we use in arrow ?